test/puppetlabs/trapperkeeper/authorization/testutils.clj

```diff
@@ -26,3 +31,9 @@
    (new-rule type path :any))
   ([type path method]
    (rules/new-rule type path method 500 "test rule")))
+
+(defn url-encoded-cert
+  [x509-cert]
+  (let [cert-writer (StringWriter.)
+        _ (ssl-utils/cert->pem! x509-cert cert-writer)]
+    (ring-codec/url-encode cert-writer)))
```

Comment:
I don't think I can do that, because `ssl-utils/cert->pem!` just returns back nil, and not the `writer`.